.env.gitpod

```diff
@@ -27,7 +27,7 @@ MAXMIND_USER_ID = 123456
 MAXMIND_LICENCE_KEY = "xxxxxxxxxxxxxxxx"
 
 MICROSOFT_PROVIDER_ID = 1234
-MICROSOFT_LOCAL_FILE_DIRECTORY = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
+MICROSOFT_REMOTE_API_ENDPOINT = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
```

Comment:
What do you think of renaming all these constants from:
`PROVIDER_REMOTE_API_ENDPOINT`
to
`PROVIDER_OS_ENDPOINT` or `PROVIDER_TGWF_OS_ENDPOINT`